cmd/gitops-pusher/gitops-pusher.go

```diff
@@ -162,7 +162,7 @@ func main() {
 		log.Fatal("set either the envvar TS_API_KEY or TS_OAUTH_ID and TS_OAUTH_SECRET")
 	}
 	var client *http.Client
-	if oiok {
+	if oiok && (oauthId != "" || oauthSecret != "") {
```

Comment:
Do we want _both_ to be non-empty, or is it OK as long as we have just one?

Reply:
Strictly speaking we just need the secret for this operation, but I would also say that if either of them are non-empty it means that the user attempted to set _something_, and if that operation fails because the other one was empty in an unexpected way we should be erroring through the oauth flow instead of the api flow.

Reply:
That might be worth noting in a lil comment.

Reply:
done, ty!